routes/salesRoutes.js

```diff
@@ -76,31 +86,59 @@ router.put("/:saleId", async (req, res, next) => {
  * Method to add sales to the database
  * @returns sends a response to the requester indicating whether or not record creation was successful
  */
-router.post("/", async (req, res, next) => {
-	const { ticket_id, sale_amount } = req.body;
-
-	if (!ticket_id || !sale_amount) {
-		res.status(400).json({ message: "All fields are required." });
-	}
-
-	if (typeof sale_amount === "string") {
-		user_id = parseFloat(user_id, 10);
-	}
-
+router.post("/", async (req, res) => {
+	console.log("Received payment request: ", req.body);
+	const { amount, description, token } = req.body;
 	try {
-		const sale = await db.addRecord("Sales", { ticket_id, sale_amount });
-
-		if (sale) {
-			res.status(201).send(sale);
-		} else {
-			res
-				.status(400)
-				.json({ message: "Something went wrong. Could not create sale." });
-		}
-	} catch (err) {
-		console.log("Internal server error: ", err);
-		res.status(500).json({ message: "Internal server error.", error: err });
+		const charge = await stripe.charges.create({
+			amount,
+			currency: "usd",
+			description,
+			source: token.id
+		});
+
+		//
+		if (charge) {
+			console.log("Success", charge);
+			res.send({ success: charge });
+		} else throw error;
+	} catch (error) {
+		console.log("ERR MSG: ", error);
+		res.status(500).send({ message: "Internal server error.", error: error });
 	}
+
+	// const body = {
```

Comment:
is there a reason we commented this out instead of just deleting it? If not then lets delete it to keep the code clean

Reply:
That was the previous code before the Stripe integration. I'll clean that up before merging the PR.